ci/scripts/python_wheel_unix_test.sh

```diff
@@ -78,6 +83,7 @@ fi
 if [ "${CHECK_UNITTESTS}" == "ON" ]; then
   # Install testing dependencies
   pip install -U -r ${source_dir}/python/requirements-wheel-test.txt
+  PYTHON=python ${source_dir}/ci/scripts/install_gcs_testbench.sh default
```

Comment:
Hmm, what is `PYTHON=python` supposed to solve here?

Reply:
We use `python3` not `python` by default in `ci/scripts/install_gcs_testbench.sh`: https://github.com/apache/arrow/blob/master/ci/scripts/install_gcs_testbench.sh#L37

Reply:
Based on the install script this will break the arm wheel builds.

Reply:
Could we call the testbench install script at the call sites of `python_wheel_unix_test.sh` instead?

Reply:
Something like the following?

```diff
diff --git a/ci/docker/python-wheel-manylinux-test.dockerfile b/ci/docker/python-wheel-manylinux-test.dockerfile
index 55c27d1d7b..cdd0ae3ced 100644
--- a/ci/docker/python-wheel-manylinux-test.dockerfile
+++ b/ci/docker/python-wheel-manylinux-test.dockerfile
@@ -25,3 +25,6 @@ FROM ${arch}/python:${python}
 # test dependencies in a docker image
 COPY python/requirements-wheel-test.txt /arrow/python/
 RUN pip install -r /arrow/python/requirements-wheel-test.txt
+
+COPY ci/scripts/install_gcs_testbench.sh /arrow/ci/scripts/
+RUN PYTHON=python /arrow/ci/scripts/install_gcs_testbench.sh default
```

I think that it's a good idea.

Reply:
I won't have access to a computer for at least 24 hours feel free to push changes if you would like to try it out.

Reply:
Done: e14345ad25